machine/wiwynn/andromeda/INSTALL

```diff
@@ -0,0 +1,116 @@
+Cross-Compiling ONIE
```

Comment:
The machine directory should be of the form:
```
machine/<vendor>/<vendor>_<machine>
```
For the machine you have defined here, it should look like:
```
machine/wiwynn/wiwynn_andromeda
```